docs/usersguide/migrating.rst

```diff
@@ -58,8 +58,19 @@ i.e.:
 
 .. code-block:: bash
 
-    cd /opt/invenio/var/data
-    tar -zcvf /vagrant/dump/files.tar.gz files
+    $ cd /opt/invenio/var/data
+    $ tar -zcvf /vagrant/dump/files.tar.gz files
+
+**Pro-tip**: Maybe you want to have different data models in your new
```

Comment:
I remember talking with @lnielsen about this, I think we are using it in other places, i.e. http://invenio.readthedocs.io/en/iugw2017/community/contribution-guide.html#commit-messages (To be 100% accurate is not Pro-Tip but ProTip)

The idea might come from GitHub documentation, i.e. https://guides.github.com/introduction/flow/